packages/components/psammead-radio-schedule/src/ProgramCard/index.jsx

```diff
@@ -153,26 +159,24 @@ const renderHeaderContent = ({
 
   const content = (
     <HeadingContentWrapper>
-      {!isOnDemand && (
+      {isLive && (
+        <LiveLabel
+          service={service}
+          dir={dir}
+          liveText={liveLabel}
+          ariaHidden={liveLabelIsEnglish}
+          offScreenText={liveLabelIsEnglish ? 'Live' : null}
+        />
+      )}
+      {isNext && (
         <>
-          <LabelWrapper
-            service={service}
-            script={script}
-            {...labelWrapperProps}
-            {...programStateConfig[state]}
-          >
-            {`${stateLabel.toUpperCase()}`}
-          </LabelWrapper>
-          <VisuallyHiddenText {...hiddenTextProps}>
-            {stateLabel === 'Live' && ` ${stateLabel}`}
-            {`,`}
-          </VisuallyHiddenText>
+          <NextLabel service={service} script={script} dir={dir}>
+            {nextLabel}
+          </NextLabel>
+          <VisuallyHiddenText>{`, `}</VisuallyHiddenText>
```

Comment:
Do we really need this visually hidden. Span with just a comma?

Reply:
I left this as was, it matches the screen reader UX but I agree (I think you're saying this!) we're adding way too much complexity across the board hiding and showing elements.

Reply:
The Live Label does not add this extra hidden comma unless it is within the visually hidden text - I will do the same here and we can decide how to proceed when we move this into its own component.